src/pretix/base/views/metrics.py

```diff
@@ -0,0 +1,35 @@
+from django.conf import settings
+from http import HttpResponse
+from .. import metrics
+
+unauthedResponse = HttpResponse("<html><title>Forbidden</title><body>You are not authorized to view this page.</body></html>", mimetype="text/html")
+unauthedResponse["WWW-Authenticate"] = 'Basic realm="metrics"'
+unauthedResponse.status_code = 401
+
+
+def serve_metrics(request):
+    # first check if the user is properly authorized:
+    if "HTTP_AUTHORIZATION" not in request.META:
+        return unauthedResponse
+
+    method, credentials = request.META["HTTP_AUTHORIZATION"].split(" ", 1)
+    if method.lower() != "basic":
+        return unauthedResponse
+
+    user, passphrase = credentials.strip().decode("base64").split(":", 1)
+
+    if user != settings.METRICS_USER:
```

Comment:
There is no way to set this yet, right? We'd need to somehow read this from the config file in ``pretix/settings.py``. Also, we'll need documentation on this in ``doc/admin/config.rst``.